samples/BucketBrigade/BucketBrigadeSample.qs

```diff
@@ -29,14 +29,21 @@
         // Generate a (Int, Bool[]) array of data.
         let data = GenerateMemoryData();
         // Create the QRAM.
-        using (memoryRegister = Qubit[(2^data::AddressSize) * data::DataSize]){
+        using (flatMemoryRegister = Qubit[(2^data::AddressSize) * data::DataSize]){
```

Comment:
What's the distinction between `flatMemoryRegister` and `memoryRegister`. Maybe a comment?

Reply:
I have added more comments and a new helper to make the structuring of the memory register easier.